lib/cassandra/custom_data.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+#--
+# Copyright 2013-2016 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#++
+
+# Use this module to mark domain object classes as custom type implementations for custom-type
+# columns in C*. This module has no logic of its own, but indicates that the marked class has
+# certain methods.
+# @private
+module Cassandra::CustomData
```

Comment:
Well, we have Types::Custom already, and CustomType would sound too similar; one might think one is an alias for the other. Also, I wanted to signify that a CustomData class does processing of data of a custom-type.